materials/projects.md

```diff
@@ -5,13 +5,40 @@ title: Projects - Intro
 language: R
 ---
 
-* R, databases or both
-* Can be on anything
-* Should be something you are excited about
-* 30-40 hours of work
+## Goals
 
-* Email me a <1 page proposal whenever you're ready or set up some time to chat
-* No later than November 1st
+Course projects are intended to give you the opportunity to engage in a larger
+project on a topic of your choosing. If you are a graduate student or an
+undergraduate researcher choosing a project that is directly related to your
```

Comment:
"If you are a graduate student or an undergraduate researcher, we encourage you to choose a project that is directly related to your research."